### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by bnmajor**
Add separate conda instructions for each platform and clarify the differences between the release options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,15 +45,30 @@ pip install --no-deps -U -e hexrd
...
+### Windows
+```bash
+# First, make sure python3.8+ is installed.
+# If it is not, run the following command:
+conda install -c anaconda python=3.8
+# Install deps using conda package
+conda install -c HEXRD -c cjh1 -c anaconda -c conda-forge hexrdgui
+# Now using pip to link repo's into environment for development
+pip install --no-deps -U -e hexrd
+BUILD_CONDA_BUILD=1
```
Comment (cjh1) [inline README.md:57]: We need `set` here

File: README.md [hunk 2]
```diff
@@ -32,15 +45,30 @@ pip install --no-deps -U -e hexrd
 CONDA_BUILD=1 pip install --no-deps -U -e hexrdgui
```
Comment (cjh1) [inline README.md:45]: We can remove the first python install, only need one!

File: README.md (hunk 1 above)
Comment (bnmajor) [reply README.md]: Sorry about that, typed the word I was looking at on accident... Fixed.

File: README.md (hunk 2 above)
Comment (bnmajor) [reply README.md:43]: Oops, overlooked that. Fixed.

**Review by cjh1 — APPROVED** [review]
Thank you!

**Pull request merged**